ArduCopter/mode.cpp

```diff
@@ -919,7 +920,7 @@ void Mode::output_to_motors()
     motors->output();
 }
 
-Mode::AltHoldModeState Mode::get_alt_hold_state(float target_climb_rate_cms)
+Mode::AltHoldModeState Mode::get_alt_hold_state_U_ms(float target_climb_rate_ms)
```

Comment:
Renaming the function is a good idea in that you'll catch all the callers; but long-term I don't thing the name here hsould have units (it's returning a ModeState thingy, not a number)

Reply:
I agree.

Reply:
I think the "long-term" plan is to remove the units from this functions name but my understanding is that this isn't a blocker so I'll mark this resolved and merge if that's OK